libs/novui/src/json-schema-components/widgets/InputEditorWidget.tsx

```diff
@@ -0,0 +1,264 @@
+import { FocusEvent, ChangeEvent, useEffect, useMemo, useRef, useState } from 'react';
+
+import { ErrorSchema, WidgetProps } from '@rjsf/utils';
+
+import { Input } from '@mantine/core';
+import { RichTextEditor } from '@mantine/tiptap';
+
+import {
+  type Extensions,
+  NodeViewContent,
+  NodeViewProps,
+  NodeViewWrapper,
+  ReactNodeViewRenderer,
+  useEditor,
+} from '@tiptap/react';
+import Text from '@tiptap/extension-text';
+import Paragraph from '@tiptap/extension-paragraph';
+import { ReactRenderer, mergeAttributes } from '@tiptap/react';
+import Document from '@tiptap/extension-document';
+import { Mention as ExternalMention } from '@tiptap/extension-mention';
+
+import { css, cx } from '../../../styled-system/css';
+import { input, inputEditorWidget } from '../../../styled-system/recipes';
+import { splitCssProps } from '../../../styled-system/jsx';
+
+import { VariableSuggestionList, SuggestionListRef, VariableItem } from './VariableSuggestionList';
+
+const inputEditorClassNames = inputEditorWidget();
+
+const AUTOCOMPLETE_OPEN_TAG = '{{';
+const AUTOCOMPLETE_CLOSE_TAG = '}}';
+
+const VALID_VARIABLE_REGEX = new RegExp(`${AUTOCOMPLETE_OPEN_TAG}([^{}]*?)${AUTOCOMPLETE_CLOSE_TAG}`, 'g');
+
+// To get initial invalid syntax of {{..} or {{..<space>
+const INVALID_VARIABLE_REGEX = new RegExp(
+  `(${AUTOCOMPLETE_OPEN_TAG}[^{}|\\s]*[}|\\s](?!})|{{[^{}]*}(?!}})|{{[^{}]*$)`,
+  'g'
+);
+const AUTOCOMPLETE_REGEX_V2 = new RegExp(`${AUTOCOMPLETE_OPEN_TAG}(.*?(.*?))${AUTOCOMPLETE_CLOSE_TAG}`, 'm');
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
✏ **suggestion:** I don't think the `_V2` suffix is necessary here.‏
```suggestion
const AUTOCOMPLETE_REGEX = new RegExp(`${AUTOCOMPLETE_OPEN_TAG}(.*?(.*?))${AUTOCOMPLETE_CLOSE_TAG}`, 'm');
```

Reply:
oops, there used to be eight of those ;) thanks!